app/src/main/res/values-v23/styles.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="AppTheme.Light.Widget" parent="AppTheme.Light">
+        <item name="colorPrimary">@color/supla_green</item>
```

Comment:
Just a suggestion to keep all styling properties in default styles.xml _unless_ it's necessary for them to have different values for specific targets. Otherwise we will end up with multiple copies of resource definitions to remember about and maintain.